frontend/src/components/Sidebar.vue

```diff
@@ -1,5 +1,20 @@
 <template>
   <nav :class="{ active }">
+    <button v-if="user.username" @click="toAccountSettings" class="action">
+      <i class="material-icons">person</i>
+      <span>{{ user.username }}</span>
+    </button>
+    <button
+      v-if="canLogout"
+      @click="logout"
+      class="action"
+      id="logout"
+      :aria-label="$t('sidebar.logout')"
+      :title="$t('sidebar.logout')"
+    >
+      <i class="material-icons">exit_to_app</i>
+      <span>{{ $t("sidebar.logout") }}</span>
+    </button>
```

Comment:
Why is it moved?

Reply:
To me it made more sense to have profile related buttons together

Reply:
Logout is not a common thing that users want to do, so it should be placed on top as a main action

Reply:
You mean should _not_ right?

Is it better if it's moved to the bottom of the sidebar?